support/ci/cargo_update.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+set -euo pipefail 
+ 
+# shellcheck source=./support/ci/shared.sh 
+source ./support/ci/shared.sh 
+
+branch="ci/cargo-update-$(date +"%Y%m%d")"
+git checkout -b "$branch"
+
+toolchain="$(get_toolchain)"
+
+install_rustup
+install_rust_toolchain "$toolchain"
+
+echo "--- :box: Cargo Update"
+cargo +"$toolchain" update
+echo "--- :box: Cargo Check"
+cargo +"$toolchain" check --quiet --all --tests
+
+git add Cargo.lock
+
+if [ "$BUILDKITE" != "true" ]; then 
+  echo "Not running in Buildkite. Please verify cargo updated correctly before opening a Pull Request" 
+  exit 0
+fi
+
+git commit -s -m "Update Cargo.lock"
+
+open_pull_request
```

Comment:
Where is this defined?

Reply:
This is defined as part of Expeditor: https://expeditor.chef.io/docs/reference/script/#open-pull-request 

I've added a comment linking to the documentation in the source :smile_cat:

Reply:
:thinking: If this is explicitly relying on Expeditor functionality, it might be better to put it under `.expeditor`.

I wouldn't expect for anything in `support/ci` to necessarily do that. As we centralize more logic in Expeditor, moving things under `.expeditor` makes more sense, too.

Reply:
100% ^^^